solutions/bio2/TickTock/index.tsx

```diff
@@ -0,0 +1,47 @@
+import { SCodeBlock, SText, STitle } from "components";
+import { brace } from "solutions/util";
+
+export const TickTock = () => {
+  return (
+    <>
+      <STitle>Idea</STitle>
+      <STitle>Abridged Statement</STitle>
+      <SText>
+      You have $n$ clocks, each of which are audible from time $f$ to $l$ inclusive and make a tick sound when they are audible and the sum of the digits of the time they display is even.
+      </SText>
+      <SText>
+      The input given is the integer $n$ and then $n$ lines of two integers $f_i$ and $l_i$, corresponding to the $f$ and $l$ values of the audible period of the $i$th clock.
+      </SText>
+      <SText>
+      Output the total number of seconds in which at least one clock is audible and the number of those seconds in which a tick occurs.
+      </SText>
+      <STitle>Solution</STitle>
+      <SText>
+        If we were to calculate the number of ticks each clock makes, this could lead to duplicates being added to the output if multiple clocks are audible at the same time. To rectify this, before doing calculations,
+        we need to merge overlapping intervals to get all the intervals in which at least one clock is audible. This can be done by sorting the $f_i, l_i$ pairs in ascending order of $f$ and comparing adjacent intervals.
+      </SText>
+      <SText>
+        The total audible period can be calculated by summing $l - f + 1$ over all merged intervals, and the number of ticks in each interval can then be calculated by subtracting the number of ticks up to $f$ exclusive from the number of ticks up to $l + 1$ exclusive.
+      </SText>
+      <SText>
+        Now for the formula to calculate ticks up to a value $n$ exclusive. Given a number with $x$ digits, the number of ticks with $n + 1$ is always $5$, since if the sum of the first $n$ digits is even, the unit digits that yield a tick are $0$, $2$, $4$, $6$, and $8$, and if the sum is odd, then the units digits that yield a tick are $1$, $3$, etc.
```

Comment:
I struggle to understand what this sentence is saying. What number has $x$ digits? Is it $n$? But then what does it mean by the number of ticks **with** $n + 1$ ? I'm a bit confused, could you please explain this to me?